CHANGELOG.md

```diff
@@ -1,6 +1,35 @@
 Change Log for libSplash
 ================================================================
 
+Release 1.7.0
+-------------
+**Date:** 2018-01-22
+
+This release adds modernized CMake scripts including CMake config
+packages on install with targets. A new `readAttributeInfo` method
+was added to the `DataCollector` for save reading of types and
+extends of attributes. On reads, the `offset` in `DataContainer` for
+non-zero offsets in domains was fixed. Due to a bug in upsteam CMake,
+at least version 3.10.0 of CMake is required now.
+
+**Interface Changes**
+
+ - new read method: attribute meta info #248 #268
+ - keep info on H5 datatypes in collection types #259
```

Comment:
QUESTION:
Why not use complete abbreviation hdf5?

Reply:
took it from the title of the PR and it's the actual name

Reply:
That's fine with me